cifer/chain.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "cifer",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://cifer.ai/",
+  "pretty_name": "Cifer",
+  "chain_id": "cifer",
+  "bech32_prefix": "cifer",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md